lib/ansible/module_utils/yumdnf.py

```diff
@@ -86,6 +87,7 @@ def __init__(self, module):
         self.releasever = self.module.params['releasever']
         self.security = self.module.params['security']
         self.skip_broken = self.module.params['skip_broken']
+        self.nobest = self.module.params['nobest']
```

Comment:
This PR appears to make use of `self.nobest` only in the `dnf` module, yet it is defined for both `yum` and `dnf`. If `--nobest` functionality is available in `dnf` only, the option would need to be added to `lib/ansible/modules/dnf.py` instead.

Reply:
Thanks, moved to `lib/ansible/modules/dnf.py`.